backend/models/Profile.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+const { InterestSchema } = require("./Interest");
+const { LocationSchema } = require("./Location");
+const { ProfileImageSchema } = require("./ProfileImage");
+
+const ProfileSchema = new mongoose.Schema({
```

Comment:
We probably also need some means of representing the user's match preferences, e.g. in the people they're matched with what they hope to find in terms of gender, age range, location, common activity interests, etc.

Reply:
That's true, we might want to think about that further when we get to implementing that, we can probably merge this and then start the API endpoints/wiring